src/modules/generateli.js

```diff
@@ -0,0 +1,38 @@
+import { editingTask } from './editingTask.js';
+import taskStatus from './taskStatus.js';
+
+function generate(item, container) {
+  const div = document.createElement('div');
+  const element = item;
+  const li = document.createElement('li');
+  container.appendChild(li);
+  const input = document.createElement('input');
+  li.appendChild(div);
+  input.setAttribute('type', 'checkbox');
+  const label = document.createElement('label');
+  div.appendChild(label);
+  div.appendChild(input);
+  input.addEventListener('change', taskStatus);
+  // text editable input-- task text
+  const taskDescription = document.createElement('input');
+  taskDescription.value = element.description;
+  div.appendChild(taskDescription);
+  taskDescription.classList.add('taskDescription');
+  // div.innerHTML += element.description;
+  const dotIcon = document.createElement('img');
+  dotIcon.setAttribute('src', './assets/images/dot.png');
+  dotIcon.classList.add('dotIcon');
+  li.appendChild(dotIcon);
+  li.classList.add(`index${element.index}`);
```

Comment:
Seems like you are using the class names later in your code as dynamic selectors, kindly please consider using [data attributes](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes) that can help you to pass data in HTML attributes, like ids and indexes.